.travis/build.sh

```diff
@@ -27,7 +27,6 @@ fi
 
 # Setup ARCH SDK paths
 if [ "$SMING_ARCH" == "Esp8266" ]; then
-	export ESP_HOME=$TRAVIS_BUILD_DIR/opt/esp-alt-sdk
 	export PATH=$PATH:$ESP_HOME/xtensa-lx106-elf/bin:$ESP_HOME/utils/
```

Comment:
Where is the ESP_HOME set? If it is not done yet then also the PATH update will not work as expected.

Reply:
Sorry, my bad. I saw that it is now moved to .travis.yml.